src/modules/swap/handlers/handlerSwap.js

```diff
@@ -83,6 +83,7 @@ class Swap {
   getTrade(tradeInfo) {
     for (const p of this.providers) {
       if (p.provider === tradeInfo.provider) return p.getTrade(tradeInfo);
+      throw new Error('This provider is not available.');
```

Comment:
This throws since get trade is also used to fetch for quotes. Please check Asana card for the issue explanation or let me know if you have questions.

Reply:
@gamalielhere Applied try catch as per Asana explanation